Formula/fastnetmon.rb

```diff
@@ -30,8 +30,8 @@ class Fastnetmon < Formula
   uses_from_macos "ncurses"
 
   on_linux do
+    depends_on "elfutils"
     depends_on "libbpf"
-    depends_on "libelf"
```

Comment:
Trying this out as:
- `libelf` is a deprecated formula we only keep for macOS usage.
- `elfutils` is what every Linux distro provides/uses (often named `libelf` in those repositories, e.g. Arch's https://archlinux.org/packages/core/x86_64/libelf/ is `elfutils`).